app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -267,7 +300,10 @@ public function getGroup($groupId = null)
     }
 
     /**
-     * {@inheritdoc}
+     * Prepare array of store groups
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc